testsuite/clash-testsuite.cabal

```diff
@@ -87,3 +96,8 @@ executable clash-testsuite
   build-depends:
     clash-testsuite
 
+executable partial-eval
+  import: basic-config
+  main-is: partial-eval.hs
+  build-depends: concurrent-supply
+
```

Comment:
Is this something we should (optionally) be running on the CI?

Reply:
No, this program just runs up to the evaluator and then spits out the unevaluated and evaluated versions of each binding in a module. In terms of CI, the core tests are very similar as they run entities from the module under test through the evaluator to compare. This is more for debugging the evaluator.